.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!-- See https://github.com/ucsb-cs156-m23/m23-10am-4-NOTES/blob/main/masterDoc.md for more info-->
```

Comment:
Keep in mind that you are no longer working on a repo that "belongs to" ucsb-cs156-m23; if your team's repo is chosen as the best of the four happycows repos, everything in the repo gets merged into the production repo.

So it isn't appropriate to make reference to anything in this repo that is specific to your quarter or team.  The one and only exception is the links to the dokku version at the very top of the README.